src/main/java/com/kodilla/ecommercee/Controller/ProductController.java

```diff
@@ -0,0 +1,73 @@
+package com.kodilla.ecommercee.Controller;
+
+import com.kodilla.ecommercee.domain.ProductDto;
+import com.kodilla.ecommercee.mapper.ProductMapper;
+import com.kodilla.ecommercee.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+
+@RestController
+@RequestMapping("v1/product")
+public class ProductController {
+
+    @Autowired
+    private ProductService service;
+    @Autowired
+    private ProductMapper productMapper;
+
+
+    @GetMapping
+    public List<ProductDto> getProducts(){
+        return new ArrayList<>();
+    }
+
+
```

Comment:
Jeśli robimy odstępy między zmiennymi/metodami to wystarczy maksymalnie jedna pusta linia. Teraz w wielu miejscach masz po 2